kernel_gateway/notebook_http/request_utils.py

```diff
@@ -12,17 +12,20 @@
 APPLICATION_JSON = 'application/json'
 TEXT_PLAIN = 'text/plain'
 
-def format_request(bundle):
+def format_request(bundle, kernel_language='no_spec'):
```

Comment:
Another instance of the magic value 'no_spec'. I've expressed my concerns about this in one of the other review comments.

Reply:
same as above.